device/src/drivers/button.rs

```diff
@@ -32,7 +32,7 @@ impl Active for ActiveLow {
     }
 }
 
-pub struct Button<P, ACTIVE = ActiveHigh>
+pub struct Button<P, ACTIVE = ActiveLow>
```

Comment:
Instead of changing the default for all boards, maybe adjust the UserButtonBx types in the bsp.